--- a/compiler/rustc_target/src/spec/targets/x86_64_unknown_linux_musl.rs
+++ b/compiler/rustc_target/src/spec/targets/x86_64_unknown_linux_musl.rs
@@ -17,8 +17,8 @@
         | SanitizerSet::THREAD;
     base.supports_fentry = true;
     base.supports_xray = true;
-    // FIXME(compiler-team#422): musl targets should be dynamically linked by default.
-    base.crt_static_default = true;
+    // melon: dynamic linking by default, like Alpine and Void (compiler-team#422): programs share the system musl
+    base.crt_static_default = false;
 
     Target {
         llvm_target: "x86_64-unknown-linux-musl".into(),
